chess-centre-app/src/components/Navigation/AuthHeader.js

```diff
@@ -20,6 +20,11 @@ function Header() {
     history.push("/");
   };
 
+  useEffect(() => {
+    if(!given_name) signOut();
```

Comment:
This is needed for when a user clicks the "Home" button when in the Confirmation screen as they're partially logged in with a user object "userConfirmed": false property.